### PR TITLE
Add the possibility to duplicate an existing case when creating a study.

### PR DESCRIPTION
**Pull request by flomillot**
Rename some variables.

Signed-off-by: Florent MILLOT <millotflo@gmail.com>

### CODE REVIEW

**Review by bendaamerahm — APPROVED** [review]

File: src/components/dialogs/create-study-dialog.js [hunk 1]
```diff
@@ -137,7 +136,7 @@ const SelectCase = () => {
  * @param {Boolean} open Is the dialog open ?
  * @param {EventListener} onClose Event to close the dialog
  */
-export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
+export const CreateStudyDialog = ({ open, onClose, providedExistingCase }) => {
```
Comment (SlimaneAmar) [inline src/components/dialogs/create-study-dialog.js:139]: `fromExistingCase` ?
Same as  back end

File: src/components/dialogs/create-study-dialog.js [hunk 2]
```diff
@@ -152,6 +151,7 @@ export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
     const caseName = useSelector((state) => state.selectedCase);
     const activeDirectory = useSelector((state) => state.activeDirectory);
     const selectedDirectory = useSelector((state) => state.selectedDirectory);
+    // Why is this in redux ??
```
Comment (SlimaneAmar) [inline src/components/dialogs/create-study-dialog.js]: Aready discussed in daily
It's not necessary (another US)

File: src/components/dialogs/create-study-dialog.js [hunk 3]
```diff
@@ -212,11 +212,11 @@ export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
         useImportExportParams(formatWithParameters);
 
     const [
-        selectedFile,
+        providedCaseFile,
```
Comment (SlimaneAmar) [inline src/components/dialogs/create-study-dialog.js:215]: Why renaming ?

File: src/components/dialogs/create-study-dialog.js (hunk 2 above)
Comment (flomillot) [reply src/components/dialogs/create-study-dialog.js]: Ok I remove the comment

File: src/components/dialogs/create-study-dialog.js (hunk 1 above)
Comment (flomillot) [reply src/components/dialogs/create-study-dialog.js:139]: Where is it in the backend ?
It's not a boolean so I don't think using the prefix "from" is appropriate...

File: src/components/dialogs/create-study-dialog.js (hunk 3 above)
Comment (flomillot) [reply src/components/dialogs/create-study-dialog.js:215]: It wasn't clear between provided, selected, etc... and wasn't easy to know what exactly are these variables.
The file is not "selected" but "provided" that's why I renamed it.

File: src/components/dialogs/create-study-dialog.js (hunk 1 above)
Comment (SlimaneAmar) [inline src/components/dialogs/create-study-dialog.js:139]: Bug fix 
Change  the prop name in the `ContentContextualMenu` component

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

File: src/components/dialogs/create-study-dialog.js (hunk 1 above)
Comment (flomillot) [reply src/components/dialogs/create-study-dialog.js:139]: Done

**Review by SlimaneAmar — APPROVED** [review]

File: src/utils/rest-api.js [hunk 4]
```diff
@@ -342,11 +342,13 @@ export function createStudy(
     studyName,
     studyDescription,
     caseName,
+    duplicateCase,
```
Comment (SlimaneAmar) [inline src/utils/rest-api.js:345]: Rename caseName by caseUuid

Comment (flomillot) [reply src/utils/rest-api.js:345]: I renamed and cleaned a bit create-study-dialog too

**Pull request merged**